src/components/WorkOrders/CloseWorkOrderByProxy.js

```diff
@@ -63,6 +66,17 @@ const CloseWorkOrderByProxy = ({ reference }) => {
         })
       }
 
+      if (startDate !== '') {
+        await frontEndApiRequest({
+          method: 'put',
+          path: `/api/workOrders/starttime`,
+          requestData: {
+            startTime: startDate,
```

Comment:
startDate in this case includes the time too correct? If yes I'd consider renaming this to startDateAndTime

Reply:
scroll down